templates/vertical-map/script/pagination.hbs

```diff
@@ -1,13 +1,14 @@
 ANSWERS.addComponent("Pagination", Object.assign({}, {
   container: "#js-answersPagination",
   onPaginate: (newPageNumber, oldPageNumber, totalPages) => {
-    if (window.parentIFrame) {
+    document.documentElement.scrollTop = 0;
+    document.body.scrollTop = 0; // Safari
```

Comment:
to clarify, this should be applied to both default and iframe page?

Reply:
yes, this scrolls the page in the default case and scrolls the iFrame in an iframe page